tensorflow/contrib/tensorrt/BUILD

```diff
@@ -264,6 +264,22 @@ tf_cc_test(
     ],
 )
 
+tf_cc_test(
+    name = "trt_lru_cache_test",
+    size = "small",
+    srcs = ["resources/trt_lru_cache_test.cc"],
+    tags = [
+        "no_windows",
+        "nomac",
+    ],
+    deps = [
+        ":trt_resources",
+        "//tensorflow/core:test",
+        "//tensorflow/core:test_main",
+    ],
+)
+
```

Comment:
Thanks for adding the test!!
I think you'll need to remove this extra blank line, as I believe buildifier only allow one blank line between rules and will fail the test before merge.